src/lib/y2storage/encryption_method.rb

```diff
@@ -34,6 +34,10 @@ class EncryptionMethod
     extend Yast::I18n
 
     # Constructor
+    #
+    # @param id [Symbol]
+    # @param label [String]
+    # @param process_class [Y2Storage::EncryptionProcesses]
```

Comment:
`process_class` is a class name. I am not sure how to specify it. Maybe, something like the following?

```
# @param process_class [#new] class name of the encryption process (e.g., EncryptionProcesses::Luks1)
```